libs/cardano-cddl/app/Main.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+
+module Main where
+
+import Codec.CBOR.Cuddle.CDDL (sortCDDL)
+import Codec.CBOR.Cuddle.Huddle (Huddle, toCDDL)
+import Codec.CBOR.Cuddle.Pretty ()
+import Data.Foldable (for_)
+import Prettyprinter (Pretty (pretty))
+import Prettyprinter.Render.Text (hPutDoc)
+import System.IO (IOMode (..), hPutStrLn, withFile)
+import Test.Cardano.Ledger.Conway.CDDL qualified as Conway
+
+-- | Message to be appended to the top of every generated file
+preface :: String
+preface =
+  "; This file was auto-generated by cuddle-cddl. Please do not modify it directly!"
+
+-- | Map from spec to the CDDL file location
+cuddleSpecs :: [(Huddle, String)]
+cuddleSpecs =
+  [ (Conway.conway, "./eras/conway/impl/cddl-files/conway-merged.cddl")
+  ]
+
+writeSpec :: Huddle -> String -> IO ()
```

Comment:
Any reusable helper functions like this can go into `cardano-ledger-binary:testlib`